app/tests/lectures/test_models.py

```diff
@@ -0,0 +1,40 @@
+import pytest
```

Comment:
I do not see tests for views and serialisers within this pull request. Please create ones.